app/src/main/java/com/stevesoltys/seedvault/restore/RestoreViewModel.kt

```diff
@@ -341,6 +372,13 @@ internal class RestoreViewModel(
          *   as a whole failed.
          */
         override fun restoreFinished(result: Int) {
+
+            // Restore next chunk if successful and there are more packages to restore.
+            if (result == 0 && packageIndex < packages.size) {
```

Comment:
If the result isn't `0`, should we still try to restore the other packages at least? Btw. is there a constant we can use in place of `0`?

Reply:
Changed to constant, keeps going onto the next chunk if we fail.